templates/includes/sidebar.html

```diff
@@ -0,0 +1,9 @@
+<div class="sidebar">
+    <h2>Sidebar</h2>
+    <ul>
+        <li><a href="#">Home page</a></li>
+        <li><a href="#">Manufacturers</a></li>
+        <li><a href="#">Cars</a></li>
+        <li><a href="#">Drivers</a></li>
```

Comment:
The links in the sidebar are currently placeholders (`#`). Consider updating these to point to the actual URLs for the home page, manufacturers, cars, and drivers within your application.